src/admin-events/admin-events.spec.js

```diff
@@ -0,0 +1,106 @@
+const ethers = require('ethers');
+const { getAddress } = require('@uma/contracts-node');
+
+// Load Config files
+const { TransactionEvent } = require('forta-agent');
+
+const chainId = 1;
```

Comment:
Same comment from admin-events.js -- we haven't been consistent about this, but chainId should probably be CHAIN_ID

Reply:
Done